platforms-serverless/serverless-framework-lambda/index.ts

```diff
@@ -1,4 +1,5 @@
-import { PrismaClient, Prisma } from '@prisma/client'
+import { PrismaClient, Prisma } from './prisma/client'
```

Comment:
Best practive of importing from `@prisma/client` not possible on Lambda via Serverless?

Reply:
Related to https://github.com/prisma/ecosystem-tests/pull/3463#discussion_r1160130018
We generate in a custom folder